apply-pr.js

```diff
@@ -58,191 +58,40 @@ var ownerRepoPr = /(.*)\/(.*)#(\d*)/i.exec(ownerRepoPrArg);
 if (!ownerRepoPr) {
   exitWithMsg('Usage example: git-apply-pr joyent/node#1337');
 }
-var OWNER = ownerRepoPr[1];
-var REPO = ownerRepoPr[2];
-var PR = +ownerRepoPr[3];
-var HOST = 'github.com';
-
-var github = new (require('github'))({
-  version: '3.0.0',
-  headers: {
-    'user-agent': 'git-apply-pr',
-  },
-});
 
-// TODO XXX FIXME This should probably handle winders
-var config = path.join(process.env.HOME, '.gitconfig');
 
-if (fs.existsSync(config)) {
-  var match = fs.readFileSync(config, 'utf8').match(/token\s*=\s*([a-f0-9]+)/);
+var config = {
+  OWNER: ownerRepoPr[1],
```

Comment:
I'm not sure the `OWNER`, `REPO` and `PR` fields belong to a `config` object, because they're not part of any configuration. They're parameters for a single invocation of `git-apply-pr`, so I would suggest separating them from the configuration.